src/main/java/com/epam/training/circularbuffer/CircularBuffer.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.training.circularbuffer;
+
+import com.epam.training.exception.NotEnoughSpaceException;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public class CircularBuffer<T> {
+    private Object[] data;
+    private int initialCapacity;
+    private int headPosition = 0;
```

Comment:
Non necessary initialization.

Reply:
Yes, it's obvious code, but plain and clear